src/casync-tool.c

```diff
@@ -87,6 +88,7 @@ static void help(void) {
                "Content-Addressable Data Synchronization Tool\n\n"
                "  -h --help                  Show this help\n"
                "     --version               Show brief version information\n"
+               "  -l --log-level=LEVEL       Set log level (debug, info, err)\n"
```

Comment:
Note that I took ownership of `-l` here, you might not want that.

Reply:
hmm, syslog calls these "priorities" in most of its specs. And tools like util-linux' "logger" and journalctl expose this as `-p | --priority=` hence. I think we should do the same here.

Reply:
I got my inspiration from systemd here:

```
$ systemd -h
...
     --log-level=LEVEL           Set log level (debug, info, notice, warning, err, crit, alert, emerg)
```

This being said, I'm fine with `-p | --priority=`, let me do the change.

Reply:
Hmm, also the `syslog` man page refers to these as levels, and defines priority as something a bit different: `The  priority  argument  is formed by ORing together a facility value and a level value (described below).`

Same goes for the `logger` man page: `The priority may be specified numerically or as a facility.level  pair.   For  example,  -p  local3.info logs the message as informational in the local3 facility. The default is user.notice.`

Are you sure that you want to use `-p`? (I'm fine with both really, just want a confirmation before doing the changes).

Reply:
> This being said, I'm fine with `-p | --priority=`, let me do the change.

Uh oh, true. That is kinda borked. :-(

Reply:
Let's stick to -l then